pydantic_ai_slim/pydantic_ai/models/bedrock.py

```diff
@@ -672,6 +673,9 @@ async def _map_user_prompt(part: UserPromptPart, document_count: Iterator[int])
                         content.append({'video': video})
                 elif isinstance(item, AudioUrl):  # pragma: no cover
                     raise NotImplementedError('Audio is not supported yet.')
+                elif isinstance(item, CachePoint):
+                    # Bedrock doesn't support prompt caching via CachePoint in this implementation
```

Comment:
I just created a new issue we can link to:

```suggestion
                    # Bedrock support has not been implemented yet: https://github.com/pydantic/pydantic-ai/issues/3418
```